.github/workflows/main.yml

```diff
@@ -50,6 +50,8 @@ jobs:
         run: make test-setup
       - name: fake redhat-uep.pem for redhat_manifest module
         run: sudo mkdir -p /etc/rhsm/ca/ && sudo touch /etc/rhsm/ca/redhat-uep.pem
+      - name: adjust inventory name
+        run: sed -i '/NAME/ s/theforeman\.foreman\.//' plugins/inventory/foreman.py
```

Comment:
Can we have this in the other direction?
- Change the file for real to make it work outside of the collection
- Compile it in the Makefile for inclusion into the collection

Reply:
Yes, we could do that. I've been avoiding it for now for the sake of "we don't actually want to support that outside the collection".

Reply:
But this step would also be necessary to run the test locally, yet it is hidden deeply in the gh workflow.